src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -166,16 +170,85 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
     };
 
     handleFetchError = (error: Object): void => {
-        const { onUpgradeClick } = this.props;
+        const { currentUser, onUpgradeClick, showUpsell, showUpsellWithPicture } = this.props;
         const shouldShowVersionErrorWithUpsell = !!onUpgradeClick && error.status === 403;
-        this.setState({
-            error: shouldShowVersionErrorWithUpsell ? messages.versionNotAvailable : messages.versionFetchError,
-            errorTitle: shouldShowVersionErrorWithUpsell ? messages.versionAccessError : messages.versionServerError,
-            isLoading: false,
-            isWatermarked: false,
-            versionCount: 0,
-            versions: [],
-        });
+        const shouldShowVersions = showUpsell || showUpsellWithPicture;
+
+        if (!shouldShowVersions) {
+            this.setState({
+                error: shouldShowVersionErrorWithUpsell ? messages.versionNotAvailable : messages.versionFetchError,
+                errorTitle: shouldShowVersionErrorWithUpsell
+                    ? messages.versionAccessError
+                    : messages.versionServerError,
+                isLoading: false,
+                isWatermarked: false,
+                versionCount: 0,
+                versions: [],
+            });
+        } else {
+            const isWatermarked = false;
+            const versionLimit = Infinity;
+            const versionCount = 3;
+            const versions = [
+                {
+                    type: 'file_version',
+                    id: '28338883546',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    modified_at: '2021-11-29T15:01:19-08:00',
+                    modified_by: currentUser,
```

Comment:
I think this experience could be very confusing to the point of causing concern, as it implies a level for access to the file that the current user may not have. It seems like a static background image with clearly mock data would be preferable for simplicity and clarity.

Reply:
Added a question here: https://github.com/box/box-ui-elements/pull/2830#discussion_r809399386

I'm curious what `very confusing to the point of causing concern` could mean here?

Reply:
My main concern with this particular line is that we're indicating the current user, who may or may not be a collaborator on the current file, uploaded a new version of the file. Version history is in some ways a "system of record," so displaying inaccurate data could lead to alarm/concern.